src/pages/Auth/Login/Login.jsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from 'react-router-dom';
+import '../auth.css';
+export const Login = () => {
+  const navigate = useNavigate();
```

Comment:
like the way you have used to navigate, instead of using "Link" tag

Reply:
thanks